### DISCUSSION TITLE
enarratives - Narratives Module

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#955] [Structural Changes]
This is a discussion to collate and discuss changes that we would like to see in the narratives in EMu. These may be structural or data.

[OPEN STRUCTURAL CHANGE ISSUES](https://github.com/fieldmuseum/EMu-Documentation/issues?q=is%3Aissue+is%3Aopen+label%3A%22structural+changes%22+label%3Aenarratives)

**Comment by rondlg** [comment]
# Purpose/Subtype

in v5.1 and v9 
- The field labelled Type on the QryDetTab is mapped to NarType_tab instead of DesType_tab. 
- DesType_tab should be added the AllDetTab below NarType_tab

**Comment by rondlg** [comment]
Standardise the use of `NarType_tab` and `NarExplanation_tab`

https://github.com/fieldmuseum/EMu-Documentation/discussions/1147

**Comment by mbrownlee2** [comment]
Can we have the SumSubtitle text from the AllSum2Tab included in the Summary string for eNarratives module records?
